common/attest_plugin.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/internal/hexdump.h>
+#include <openenclave/internal/print.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/safemath.h>
+#include <openenclave/internal/utils.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "common.h"
+
+#include <openenclave/internal/plugin.h>
+
+/**
+ * Evidence header: the structure that the OE SDK runtime puts on top of
+ * evidence data.
```

Comment:
The structure should be up to the plugin not "the OE SDK runtime".  We have to be able to accommodate various evidence formats, so you cannot dictate what needs to go over the wire, for all formats.

Reply:
As discussed in today's SIG-attestation meeting, this header structure is part of the existing implementation. Will be addressed as part of PR #2999